test/Data/Map/JustifiedSpec.hs

```diff
@@ -41,9 +41,9 @@ spec = do
     it "wraps the underlying Data.Map.Map without modification" $ do
       withMap letters theMap == letters
 
-    it "can create singleton maps" $ property $ do
-      \(k :: Int, v :: Int) -> withSingleton k v (uncurry lookup) == v
```

Comment:
This first test's result is ignored. You could return `False` here and the test suite would pass.